client/booksindex.js

```diff
@@ -0,0 +1,67 @@
+let book;
+
+const updateBook = async (id) => {
+  const response = await fetch(`http://localhost:8080/book/${id}`);
+
+  const book = await response.json();
+
+  console.log(book);
+
+  const { title, author, isbn, format } = book;
+
+  // Filling information about the book in the form inside the modal
+  document.getElementById("isbn").value = isbn;
+  document.getElementById("title").value = title;
+  document.getElementById("author").value = author;
+  document.getElementById("format").value = format;
+
+  // Setting up the action url for the book
+  document.getElementById("form").action = `http://localhost:8080/book/${id}`;
+};
+
+const deleteBook = async (id) => {
+  const response = await fetch(`http://localhost:8080/book/${id}`, {
+    method: "DELETE",
+  });
+
+  const data = await response.json();
+  alert(`Book was deleted from my library`);
```

Comment:
It would be good to have an example of how to check the response. Maybe something like this:

```
alert(data.message);
if (response.ok) {
  location.reload();
}
```